arviz/data/io_numpyro.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=cyclic-import
```

Comment:
is this needed if pylintrc has this already?

Reply:
Its not. I was having issues with pylint so i just globally disabled circular import, which I'm not sure is a good idea. Could use feedback if its a stupid idea